### DISCUSSION TITLE
Overriding existing style

### DISCUSSION THREAD

**Discussion by ronf** [Textualize/rich#871] [Q&A] [answered]
If I have an existing Text object (or some other renderable), is there a way to override an existing style on it? For instance, let's say that I have a `Text.from_markup("[bold red]some text[/]")` and I want to disable the bold output.

I've tried passing a style on the print() of this object, but the style set on the Text object seem to take precedence, with anything on the print() only being able to set additional styles which were not previously set.

The specific use case here is that I want to accept markup text from an external source but then force certain styles on that input (such as disabling blinking) before rendering it.

**Comment by willmcgugan** [comment, marked as answer]
You can negate styles, which may have the effect you are looking for. Something like this:

```python
t = Text.from_markup("[bold red]some text[/]")
t.stylize("not bold")
```

**Reply by ronf** [reply]
I tried a few different variants that involved stylize(), but apparently I didn't get them quite right. This does exactly what I was looking for. Thanks very much!
